### DISCUSSION TITLE
First PandasAI live event 🔴

### DISCUSSION THREAD

**Discussion by gventuri** [sinaptik-ai/pandas-ai#163] [Announcements]
We are organizing a live event for next week for all the contributors and the PandasAI enthusiastic where we'll discuss about the new developments of PandasAI!

We are trying to find the best time that works for most of us. You can share your preferences here:
https://www.when2meet.com/?20164984-G5sNc

UPDATE: the event will take place Wednesday May 31st at 2pm CET! Check out the invite: https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MzE4cWR2cTBhNmhzZGJ2YjI1Z2tjOG45ZmcgbGVsZS52ZW50dXJpQG0&tmsrc=lele.venturi%40gmail.com

**Comment by Viddesh1** [comment]
Thanks for organising live event.

Will the session be recorded for future references and push on platforms like YouTube etc?

**Reply by gventuri** [reply]
@Viddesh1 yes, we'll try to record it for future references :)

**Comment by gventuri** [comment]
The event will be Wednesday may 31st at 2pm CET!

Feel free to share below any topic you'd like to discuss about during the event so that we have an agenda for that!

**Comment by gventuri** [comment]
Here's the link of the invite for the event: https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MzE4cWR2cTBhNmhzZGJ2YjI1Z2tjOG45ZmcgbGVsZS52ZW50dXJpQG0&tmsrc=lele.venturi%40gmail.com
